chi2/chinotype.py

```diff
@@ -98,7 +98,11 @@ def __init__(self, listargs=[], args={}):
         self.crc_user = db['crc_user']
         self.crc_service =  db['crc_service_name']
         self.crc_pw = db['crc_pw']
+        self.branchnodes = db['chi_branchnodes']
+        self.allbranchnodes = db['chi_allbranchnodes']
+        self.termtable = db['chi_termtable']
         self.schema = db['schema']
+        self.chischemes = db['chischemes']
```

Comment:
Config variable saying where to look for the chinotype-specific lookup table for prefixes and human-readable names if one exists, and what to name a newly-created one if it doesn't exist.